web/client-api/src/main/java/io/deephaven/web/client/api/JsPartitionedTable.java

```diff
@@ -283,15 +271,41 @@ public Column[] getColumns() {
      *
      * @return Promise of a Table
      */
-    @JsMethod
+    @Deprecated
```

Comment:
Don't remove the `@JsMethod` from this, that changes how it's output in the JSAPI. We just want to deprecate the existing function.

EDIT: Nevermind, `@JsMethod` unnecessary since this is already a `JsType`. https://www.gwtproject.org/doc/latest/DevGuideCodingBasicsJsInterop.html